dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -8,9 +8,9 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "Un thème est un fond d'écran et un ensemble de sons, icones et autres éléments vous permettant de personnaliser votre ordinateur en un clic.", IDC_STATIC, 5, 5, 235, 30
```

Comment:
```suggestion
    LTEXT "Un thème est un fond d'écran et un ensemble de sons, icônes et autres éléments vous permettant de personnaliser votre ordinateur en un clic.", IDC_STATIC, 5, 5, 235, 30
```
(typo in "icônes")

Reply:
Same here. Do that in your own PR. I didn't change that, and I don't intend to change it within this PR.